docs/Content/HideNonSystemProcesses.md

```diff
@@ -0,0 +1,17 @@
+# Hide Non System Processes
+
+HideNonSystemProcesses is a sandbox setting in [Sandboxie Ini](SandboxieIni.md).
+
+```
+   .
+   .
+   .
+   [DefaultBox]
+   HideNonSystemProcesses=y
+```
+
+Use the 'HideNonSystemProcesses=y' option to not allow sandboxed processed see processes running outside any boxes.
```

Comment:
```suggestion
Use the 'HideNonSystemProcesses=y' option to prevent sandboxed processed from seeing processes running outside the sandbox.
```

Reply:
Maybe not the case?
<img width="716" alt="image" src="https://github.com/user-attachments/assets/73152a5f-c51c-43ff-bd8f-f396d73c5c96">
(with `HideSbieProcesses=y` and `HideOtherBoxes=n` the process in one box can see process in another box, so not "outside *the* sandbox"?)

Reply:
Also the text in GUI says "outside any boxes"

Reply:
@offhub

Reply:
https://github.com/sandboxie-plus/sandboxie-docs/pull/187#discussion_r1765414989

@DavidXanatos